doc/users/config.sample

```diff
@@ -1,7 +1,7 @@
 ##                                                                       ##
 #                      Example configuration file                         #
 #                                                                         #
-#        See doc/readme.html and doc/users/cmake.html for further         #
+#        See doc/readme.html and doc/users/cmake_dealii.html for further         #
```

Comment:
@tamiko must have been in a rush when he reviewed this. His eyes would normally begin to twitch when there is some misalignment of characters floating about the screen. Would you mind please centring this on the line and make the hash symbols align?

Reply:
good catch :) I modified it!

Reply:
Almost centered! :stuck_out_tongue_closed_eyes:

Reply:
We'll merge it with the block not fully centered so that everyone with OCD reading the file will scream in horrow.

Reply:
Ups! I just centered in #9879!